docs/data/date-pickers/localization/LocalizedDatePicker.js

```diff
@@ -3,30 +3,36 @@ import frLocale from 'date-fns/locale/fr';
 import ruLocale from 'date-fns/locale/ru';
 import deLocale from 'date-fns/locale/de';
 import enLocale from 'date-fns/locale/en-US';
+import arSaLocale from 'date-fns/locale/ar-SA';
+import Stack from '@mui/material/Stack';
 import ToggleButton from '@mui/material/ToggleButton';
 import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
 import TextField from '@mui/material/TextField';
 import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
 import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { TimePicker } from '@mui/x-date-pickers/TimePicker';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 
 const localeMap = {
   en: enLocale,
   fr: frLocale,
-  ru: ruLocale,
   de: deLocale,
+  ru: ruLocale,
+  ar: arSaLocale,
 };
 
-const maskMap = {
+const dateMaskMap = {
   fr: '__/__/____',
   en: '__/__/____',
   ru: '__.__.____',
   de: '__.__.____',
+  ar: '',
 };
 
 export default function LocalizedDatePicker() {
   const [locale, setLocale] = React.useState('ru');
-  const [value, setValue] = React.useState(new Date());
+  const [dateValue, setDateValue] = React.useState(new Date());
```

Comment:
Can we name them `datePickerValue` and `timePickerValue` ?
At first I thought we were storing a time and a date apart from one another, not that it was for two distincts pickers.